app/views/application/scheme/chooseyourlocations.scala.html

```diff
@@ -20,14 +20,29 @@
     }
 }
 
-@main_template(title = "Where could you work from?", pageForms = Seq(locationForm), None, List("location-finder.js")) {
+@main_template(title = "Choose your preferred locations?", pageForms = Seq(locationForm), None, List("location-finder.js")) {
 
     @helper.form(action = routes.SchemeController.submitLocations(), 'novalidate -> "novalidate") {
         @helper.CSRF.formField
-        <h1 class="heading-xlarge">Where could you work from?</h1>
-
         <input type="text" id="hiddenInput" class="visuallyhidden">
         <div class="grid-wrapper">
+            <div class="grid grid-2-3">
+                <h1 class="heading-xlarge" id="choosePrefLocFramHeading">Choose your preferred locations</h1>
```

Comment:
Hmm taken from prototype as is and hence the typo .. let me remove id if not required